### ISSUE TITLE
[feature] Custom Readme for a Recipe

### ISSUE THREAD

**Issue by FreePhoenix888** [conan-io/conan#15667]
### What is your suggestion?

I have met such a problem that imgui's recipe documentation shows me a few lines that I should have in my conanfile.txt and CMakeLists.txt but actually I need a lot more things to do in these files. An author should have ability to have custom readme (as in npm) to tell such things

### Have you read the CONTRIBUTING guide?

- [X] I've read the CONTRIBUTING guide

**Comment by FreePhoenix888** [comment]
Related: https://github.com/conan-io/examples/issues/118#issuecomment-1943138027

**Comment by memsharded** [comment]
Thanks for your suggestion @FreePhoenix888 
Quick question: You mean in the web site https://conan.io/center for packages?

**Comment by FreePhoenix888** [comment]
> Thanks for your suggestion @FreePhoenix888 Quick question: You mean in the web site https://conan.io/center for packages?

Yes. Here https://conan.io/center/recipes/imgui?version=cci.20230105%2B1.89.2.docking is only standart info but actually a lot more is required to setup imgui library, does author has ability to have custom readme

For example zlib in npm has ability to have its own readme to provide more information to users: https://www.npmjs.com/package/zlib
